.github/workflows/integration_test.yml

```diff
@@ -6,11 +6,11 @@ on:
 
 jobs:
   lila:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-24.04
```

Comment:
Thanks for the fix! Curious about not fixing latest here, it’s to be future-proof?

Reply:
That one's probably not necessary (it looks like the runner was already using 24.04.1). It removed the warning `ubuntu-latest pipelines will use ubuntu-24.04 soon`